torch/masked/maskedtensor/core.py

```diff
@@ -0,0 +1,556 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+import warnings
+
+import torch
+from torch._masked import _sparse_coo_where, _sparse_csr_where
+from torch.overrides import get_default_nowrap_functions
+
+
+__all__ = [
+    "MaskedTensor",
+    "is_masked_tensor",
+    "get_default_nowrap_functions",
+]
+
+
+def is_masked_tensor(a):
```

Comment:
Any reason why users are not recommended to use isinstance() instead of this?

Reply:
Not in particular, I suppose just more readable?